caddy.go

```diff
@@ -836,7 +839,7 @@ var (
 	instancesMu sync.Mutex
 )
 
-const (
+var (
```

Comment:
[ Quoting <notifications@github.com> in "Re: [mholt/caddy] Implement missing..." ]
>> @@ -836,7 +839,7 @@ var (
>>  	instancesMu sync.Mutex
>>  )
>>
>> -const (
>> +var (
>
>Remember that Caddy can (soon) be compiled with multiple server types built in
>- the HTTP server will look for Caddyfile by default but looking for Corefile
>in that case just because it's built with the DNS server plugged in would be
>confusing. 

Ok, but this is a regression for me. How can we/I make it so that if -type=dns
is will look for a Corefile instead of a Caddyfile?

>Instead of changing this, have your server type specify default Caddyfile contents.

I don't have a sane default for an empty Caddyfile. I prefer to not do anything
at that point, i.e. don't start at all.

/Miek

--
Miek Gieben